examples/closures/closures.metatags

```diff
@@ -0,0 +1,2 @@
+description: A closure in Ballerina is an inner anonymous function that has visibility to the scope of its enclosing environment.
```

Comment:
Need to rethink this. Not exactly the correct description of a closure IMO

Reply:
+1 the description + example should also be updated right, maybe add a comment to that Anjana's spread sheet.

Reply:
How about something like the following?
```suggestion
description: In Ballerina, a closure is created to capture the variables of the enclosing environment when a locally defined construct (e.g., a local anonymous function) refers to them.
```